src/game_events/pump.cpp

```diff
@@ -173,7 +173,7 @@ namespace { // Support functions
 	/**
 	 * Returns true iff the given event passes all its filters.
 	 */
-	bool t_pump::filter_event(const event_handler& handler, const queued_event& ev)
+	bool pump::filter_event(const event_handler& handler, const queued_event& ev)
```

Comment:
I can do that, but it's in the `game_events` namespace so it would feel kinda redundant...